src/test/scala/mesosphere/marathon/api/validation/V2AppDefinitionValidatorTest.scala

```diff
@@ -1,234 +0,0 @@
-package mesosphere.marathon.api.validation
```

Comment:
Hey @alexanderweber sorry that I did not mention that before: why did you remove the whole test class here? Would it not make sense to use the new validation logic here as well?